### PR TITLE
update azure_rm_acs document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_acs.py
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by yuwzho — CHANGES_REQUESTED** [review]
We don't need to add documentation in [azure doc](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/doc_fragments/azure.py) and [azure tag](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/doc_fragments/azure_tags.py), because the fragment can import them

**Comment by Fred-sun** [comment]
@yuwzho Thanks for your review!  I'm going to remove the parameters associated with these two modules.

File: lib/ansible/modules/cloud/azure/azure_rm_acs.py [hunk 1]
```diff
@@ -42,6 +42,7 @@
     orchestration_platform:
         description:
             - Specifies the Container Orchestration Platform to use. Currently can be either DCOS, Kubernetes or Swarm.
+            - If specifies the Container Orchestration Platform with Kubernetes, need specify the service_principal
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_acs.py]: `I(service_principal) must be  defined if set to C(Kubernetes).`
Also add one line to service_principal description `Required when I(orchestration_platform=Kubernetes).`

The symbols definition you can find at https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation . Please also check these symbols in the documentation. I believe there are many wrong use.

**Review by yuwzho — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/azure/azure_rm_acs.py (hunk 1 above)
Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_acs.py]: Thanks yuwzho. Has changed according to comments!

**Comment by Fred-sun** [comment]
@yuwzho Please help give a review when you're available! Thanks you very much!

File: lib/ansible/modules/cloud/azure/azure_rm_acs.py [hunk 2]
```diff
@@ -18,16 +18,16 @@
 version_added: "2.4"
 short_description: Manage an Azure Container Service Instance (ACS).
 description:
-    - Create, update and delete an Azure Container Service Instance.
+    - Create, update and delete an Azure Container Service Instance (ACS).
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_acs.py]: no Instance
refer: https://docs.microsoft.com/en-us/azure/container-service/

File: lib/ansible/modules/cloud/azure/azure_rm_acs.py [hunk 3]
```diff
@@ -41,7 +41,8 @@
             - Valid azure location. Defaults to location of the resource group.
     orchestration_platform:
         description:
-            - Specifies the Container Orchestration Platform to use. Currently can be either DCOS, Kubernetes or Swarm.
+            - Specifies the Container I(orchestration_platform) to use. Currently can be either C(DCOS), C(Kubernetes) or C(Swarm).
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_acs.py]: no I(orchestration_platform)

here talks about property rather than option

**Review by yuwzho — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/azure/azure_rm_acs.py (hunk 3 above)
Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_acs.py]: Thanks yuwzho. Has changed according to comments!

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_acs.py [hunk 4]
```diff
@@ -18,7 +18,7 @@
 version_added: "2.4"
 short_description: Manage an Azure Container Service Instance (ACS).
 description:
-    - Create, update and delete an Azure Container Service Instance (ACS).
+    - Create, update and delete an Azure Container Service Instance.
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_acs.py]: Create, update and delete an Azure Container Service (ACS) instance.

**Review by yuwzho — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/azure/azure_rm_acs.py (hunk 4 above)
Comment (yuwzho) [reply lib/ansible/modules/cloud/azure/azure_rm_acs.py]: The term is "Azure Container Service", "instance" is not in the term, no need to capitalize it

Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_acs.py]: Thanks Yuwzho!

File: lib/ansible/modules/cloud/azure/azure_rm_acs.py [hunk 5]
```diff
@@ -27,7 +27,7 @@
         required: true
     name:
         description:
-            - Name of the Container Services instance.
+            - Name of the Azure Container Services instance (ACS).
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_acs.py]: Name of the Azure Container Services(ACS) instance.

File: lib/ansible/modules/cloud/azure/azure_rm_acs.py [hunk 6]
```diff
@@ -18,7 +18,7 @@
 version_added: "2.4"
 short_description: Manage an Azure Container Service Instance (ACS).
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_acs.py]: Manage an Azure Container Service(ACS) instance.

File: lib/ansible/modules/cloud/azure/azure_rm_acs.py [hunk 7]
```diff
@@ -18,7 +18,7 @@
...
 description:
-    - Create, update and delete an Azure Container Service Instance.
+    - Create, update and delete an Azure Container Service instance.
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_acs.py]: Azure Container Service(ACS)

**Review by yuwzho — CHANGES_REQUESTED** [review]

**Review by yuwzho — APPROVED** [review]

**Comment by yuwzho** [comment]
shipit

**Pull request merged**